enhancements/authentication/configuring-webhook-token-authenticators.md

```diff
@@ -0,0 +1,159 @@
+---
+title: configuring-webhook-token-authenticators
+authors:
+  - "@stlaz"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+approvers:
+  - "@deads2k"
+  - "@sttts"
+  - "@mfojtik"
+creation-date: 2019-12-10
+last-updated: 2019-12-10
+status: implementable
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Configuring Webhook Token Authenticators
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+  1. Should we leverage the `webhookTokenAuthenticators` field of the `authentications.config.openshift.io`
```

Comment:
This is a CRD. Yes, I think we should use it.  Simple, directly kube-compatible, not appreciably harder for us. Not expected to manually managed.  List all those reasons for the choice.